pkg/reconciler/pubsub/reconciler.go

```diff
@@ -61,15 +61,23 @@ const (
 	OpsJobOngoing            OpsJobStatus = "JOB_ONGOING"
 )
 
-func (c *PubSubBase) EnsureSubscription(ctx context.Context, owner kmeta.OwnerRefable, project, topic, subscription string) (OpsJobStatus, error) {
+func (c *PubSubBase) EnsureSubscriptionExists(ctx context.Context, owner kmeta.OwnerRefable, project, topic, subscription string) (OpsJobStatus, error) {
```

Comment:
Where is this function used, or is that in future PRs?

Reply:
I do not use it at the moment, I added it for symmetry to topic as this is a base lib.